gnupg.py

```diff
@@ -1285,6 +1314,10 @@ def delete_keys(self, fingerprints, secret=False, passphrase=None,
             fingerprints = [no_quote(s) for s in fingerprints]
         else:
             fingerprints = [no_quote(fingerprints)]
+
+        if exclamation_mode:
+            fingerprints = [f"{f}!" for f in fingerprints]
```

Comment:
Please change `f"{f}!"` to `f + '!'` because Python 2.7 is still supported, as of now, at least.

Reply:
Or `'%s!' % f`